scripts/syntax_gen.py

```diff
@@ -729,6 +721,126 @@ class SyntaxNode;
     outf.write("\n}\n")
 
 
+def generateSyntaxClone(builddir, alltypes, kindmap):
+    # Start the clone source file.
+    clonef = open(os.path.join(builddir, "AllSyntaxClone.cpp"), "w")
```

Comment:
Minor nit, but can you name this just SyntaxClone.cpp? The "All" part is unnecessary I feel.